examples/capi-quick-start/nutanix-cluster-cilium-crs.yaml

```diff
@@ -317,6 +326,18 @@ spec:
           cni:
             provider: Cilium
             strategy: ClusterResourceSet
+          csi:
+            defaultStorage:
+              providerName: nutanix
+            providers:
+            - credentials:
+                name: ${CLUSTER_NAME}-pe-creds-for-csi
+              name: nutanix
+              storageClassConfig:
+              - name: nutanix-volume
+                parameters:
+                  storageContainer: ${NUTANIX_STORAGE_CONTAINER_NAME}
+              strategy: HelmAddon
```

Comment:
```suggestion
              strategy: ClusterResourceSet
```

Reply:
We don't support ClusterResourceSet for this addon so it will always be HelmAddon.